gui-wm/wayfire/wayfire-0.7.3.ebuild

```
@@ -0,0 +1,106 @@
+# Copyright 2019-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit meson toolchain-funcs
+
+DESCRIPTION="compiz like 3D wayland compositor"
+HOMEPAGE="https://github.com/WayfireWM/wayfire"
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/WayfireWM/${PN}.git"
+else
+	SRC_URI="https://github.com/WayfireWM/${PN}/releases/download/v${PV}/${P}.tar.xz"
+	KEYWORDS="amd64 ~arm64 ~riscv ~x86"
+fi
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="debug +gles +system-wfconfig +system-wlroots X"
+
+DEPEND="
+	dev-libs/libevdev
+	dev-libs/libinput
+	dev-libs/wayland
+	gui-libs/gtk-layer-shell
+	media-libs/glm
+	media-libs/mesa:=[gles2,wayland,X?]
+	media-libs/libglvnd[X?]
+	media-libs/libjpeg-turbo
+	media-libs/libpng
+	media-libs/freetype:=[X?]
+	x11-libs/libdrm
+	x11-libs/gtk+:3=[wayland,X?]
+	x11-libs/cairo:=[X?,svg]
+	x11-libs/libxkbcommon:=[X?]
+	x11-libs/pixman
+	X? (
+		x11-base/xwayland
+		x11-libs/libxcb
+	)
+	system-wfconfig? (
+		>=gui-libs/wf-config-0.7.1
+		<gui-libs/wf-config-0.8.0
+	)
+	!system-wfconfig? ( !gui-libs/wf-config )
+	system-wlroots? (
+		>=gui-libs/wlroots-0.15.0:=[X?]
+		<gui-libs/wlroots-0.16.0:=
```

Maybe this'd be simpler if you just dependend on the subslot?
```suggestion
		gui-libs/wlroots:0/15
```

---

Thanks, applied, readding [X?] though.